.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!-- Thanks for sending a pull request!
+
+If this is your first contribution, please review and sign our contributor agreement -
+https://www.elastic.co/contributor-agreement.
+
+See also https://github.com/elastic/apm-server/blob/master/CONTRIBUTING.md for more tips on contributing.
+-->
+
+<!--
+Replace this comment with a description of what's being changed by this PR.
```

Comment:
I'd ask for a description (what) and a motivation (why). My experience is the more structured the templates are the more information one provides.